### PR TITLE
Episode 16

### PR DESCRIPTION
**Pull request by germangamboa95**

### CODE REVIEW

File: content/20211219-dev-meetups.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+---
+title: "Building a software developer community"
+slug: "building_a_software_developer-community"
+cover: "016_dev_meetups.png"
```
Comment (vincentntang) [inline content/20211219-dev-meetups.md]: could you change the cover tag?

Likewise the "slug" needs to be `-` dashes 

Example from last one 

```
title: "Javascript Features To Use in 2021"
slug: "javascript-features"
cover: "code_chefs_podcast_art.png"
date: 2021-12-19
audioPath: 015_javascript_features.mp3
episodeNumber: 15
fileSize: 124
showLength: 54:16
category: "tech"
# shortDescription is only for codechefs.dev homepage
shortDescription: "JavaScript has taken over the world over the past decade. Come learn about its history and the latest features."
tags:
  - programming
  - javascript
```

Comment (vincentntang) [reply content/20211219-dev-meetups.md]: use `cover: "code_chefs_podcast_art.png"`

And slug make it hypenated

File: content/20211219-dev-meetups.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+date: 2021-10-28
+audioPath: 016_dev_meetups.mp3
+episodeNumber: 15
```
Comment (vincentntang) [inline content/20211219-dev-meetups.md]: should be episode 16

File: content/20211219-dev-meetups.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
+---
+title: "Building a software developer community"
+slug: "building_a_software_developer-community"
+cover: "016_dev_meetups.png"
+date: 2021-10-28
```
Comment (vincentntang) [inline content/20211219-dev-meetups.md:5]: adjust the date here I think?

Comment (germangamboa95) [reply content/20211219-dev-meetups.md:5]: Do we want the original recording date there or the release date?

Comment (vincentntang) [reply content/20211219-dev-meetups.md:5]: ive been using the record date, so I can see the context of when was td about around that time

usually get the record date from zencastr mp3 creation

Comment (vincentntang) [reply content/20211219-dev-meetups.md:5]: i think 10-28 sounds right though actually

**Pull request merged**